README.md

```diff
@@ -143,7 +143,7 @@ Validator                               | Description
 **isLicensePlate(str, locale)**     | check if the string matches the format of a country's license plate.<br/><br/>`locale` is one of `['cs-CZ', 'de-DE', 'de-LI', 'en-IN', 'es-AR', 'hu-HU', 'pt-BR', 'pt-PT', 'sq-AL', 'sv-SE']` or `'any'`.
 **isLocale(str)**                       | check if the string is a locale.
 **isLowercase(str)**                    | check if the string is lowercase.
-**isLuhnValid(str)**                    | check if the string passes the [Luhn check][Luhn Check].
+**isLuhnNumber(str)**                    | check if the string passes the [Luhn algorithm check](https://en.wikipedia.org/wiki/Luhn_algorithm).
```

Comment:
```suggestion
**isLuhnNumber(str)**                    | check if the string passes the [Luhn algorithm check][Luhn Check].
```
Line 300 already makes sure it links to that wikipedia page so this should be fine

Reply:
Doesn't affect the user so we can fix in a later PR as well